anycloud/node/src/datastore.ts

```diff
@@ -30,65 +34,94 @@ export class DataStore {
     });
     return await response.text();
   }
+
   async get(dsKey: string): Promise<string> {
-    if (!dsKey) return undefined;
+    if (DataStore.isInvalidKey(dsKey)) return undefined;
     if (this.isLocal) {
       if (dsKey in this.localDS) {
         const val = this.localDS[dsKey];
-        return JSON.stringify(val);
+        return JSON.parse(val) || val;
       } else {
         return undefined;
       }
     }
     try {
-      return JSON.stringify(await this.request('GET', `${this.ctrlPortUrl}get/${dsKey.toString()}`));
+      const dsValue = await this.request('GET', `${this.ctrlPortUrl}get/${dsKey}`);
+      return JSON.parse(dsValue) || dsValue;
```

Comment:
This won't work like you want. `JSON.parse("false") || "false"` will return `"false"` instead of the actual boolean.

Reply:
Yes, I've added the try/catch